ui/packages/shared/profile/src/GraphTooltipArrow/Content.tsx

```diff
@@ -86,31 +79,15 @@ const GraphTooltipArrowContent = ({
     timeoutHandle = setTimeout(() => setIsCopied(false), 3000);
   };
 
-  const prevValue = cumulative - diff;
-  const diffRatio = diff !== 0n ? divide(diff, prevValue) : 0;
-  const diffSign = diff > 0 ? '+' : '';
-  const diffValueText = diffSign + valueFormatter(diff, unit, 1);
-  const diffPercentageText = diffSign + (diffRatio * 100).toFixed(2) + '%';
-  const diffText = `${diffValueText} (${diffPercentageText})`;
-
-  const name = nodeLabel(table, row, level, false);
-
-  const getTextForCumulative = (hoveringNodeCumulative: bigint): string => {
-    const filtered =
-      totalUnfiltered > total
-        ? ` / ${(100 * divide(hoveringNodeCumulative, total)).toFixed(2)}% of filtered`
-        : '';
-    return `${valueFormatter(hoveringNodeCumulative, unit, 2)}
-    (${(100 * divide(hoveringNodeCumulative, totalUnfiltered)).toFixed(2)}%${filtered})`;
-  };
+  const {name, locationAddress, cumulativeText, diffText, diff, row: rowNo} = graphTooltipData;
```

Comment:
Super minor comment, but could we rename "rowNo" to "rowNumber". "No" can be interpreted in multiple ways.

Reply:
Done, addressed it.